### PR TITLE
Initial setup for dev docs

### PR DESCRIPTION
**Pull request by hs-saunved**
Adds the initial setup files for the dev docs.

### CODE REVIEW

File: package.json
Comment (helpshift-ps) [inline package.json:3]: When are we going to update the version? Do we have automation to increase the version with every release? Or is that something we are planning to do?

File: package.json
Comment (helpshift-ps) [inline package.json:11]: What does clear do? Is it the same as the "clean" task in our Makefile?

Comment (hs-saunved) [reply package.json:11]: This is a CLI command given to us by Docusaurus (https://docusaurus.io/docs/cli#docusaurus-clear-sitedir). As far as I know, it gets rid of the local `.docusaurus` folder. It's not similar to the clean task in our Makefile because I don't think it gets rid of node_modules.

File: package.json
Comment (hs-saunved) [reply package.json:3]: We don't have an automation yet for bumping the version. We could use something like this: https://github.com/marketplace/actions/bumpr-bump-version-when-merging-pull-request-with-specific-labels, which is a Github Action to autobump the version under specific conditions. I'll explore this further.

File: package.json
Comment (nrkin) [inline package.json:1]: Should we add an author as well?

**Review by nrkin — COMMENTED** [review]
One comment

File: package.json
Comment (hs-saunved) [reply package.json:1]: Yes, we should do that. I can add "Helpshift" as the author.

Comment (nrkin) [reply package.json:1]: Cool

Comment (hs-saunved) [reply package.json:1]: Keeping this conversation open until I submit a commit to solve it

File: package.json
Comment (helpshift-ps) [reply package.json:3]: Please add a backlog task for this and resolve this comment.

File: package.json
Comment (helpshift-ps) [reply package.json:11]: Ack

**Review by helpshift-ps — APPROVED** [review]

File: package.json
Comment (hs-saunved) [reply package.json:3]: Done

File: package.json
Comment (hs-saunved) [reply package.json:1]: Added author and license

**Comment by hs-saunved** [comment]
Ready to merge (after furthest branch is merged)

**Pull request merged**
